apps/server/src/schema/integrations/insights-operations.ts

```diff
@@ -262,16 +262,19 @@ const GroupedInsightDto = builder.simpleObject(
       type: IFrameDTO,
       nullable: true,
       resolve: async (root, _args, _ctx, _info) => {
-        if (!root.adId) return null;
+        if (!root.adId && !root.creativeId) return null;
```

Comment:
```suggestion
        if (!root.adId || !root.creativeId) return null;
```
I haven't tested it, but shouldn't it be || instead of &&?